acceptance/pre-suite/01_install_rototiller.rb

```diff
@@ -8,8 +8,9 @@
   end
   scp_to(sut, gem_name, gem_name)
 
+  rake_version = `rake --version`.split[2]
   if ENV['RAKE_VER']
```

Comment:
good point.  but i think we want that there so it'll just install whatever version of rake on the SUT if we aren't using rake at all, locally, for instance.